src/main/java/frc/robot/RobotContainer.java

```diff
@@ -153,103 +158,100 @@ public RobotContainer() {
-                      OI.getAxisSupplier(OI.Driver.xTranslationAxis).get(),
-                      OI.getAxisSupplier(OI.Driver.yTranslationAxis).get(),
-                      OI.getAxisSupplier(OI.Driver.rotationAxis).get()),
-                  0.1);
-      drivetrain.setDefaultCommand( // Drivetrain will execute this command periodically
-          drivetrain.fieldOrientedDrive(input).withName("Get Axis Suppliers"));
-
-      OI.getButton(OI.Driver.brakeButton)
-          .whileTrue(
-              drivetrain
-                  .applyRequest(() -> new SwerveRequest.SwerveDriveBrake())
-                  .withName("Brake Swerve"));
-      OI.getTrigger(OI.Driver.pointForward)
-          .toggleOnTrue(drivetrain.pointAtLocation(new Translation2d(16.4846, 4.1), input));
-      OI.getButton(OI.Driver.resetRotationButton)
-          .onTrue(
-              drivetrain
-                  .runOnce(
-                      () ->
-                          drivetrain.seedFieldRelative(
-                              new Pose2d(
-                                  drivetrain.getState().Pose.getTranslation(),
-                                  Rotation2d.fromDegrees(180))))
-                  .withName("Put Pose & Rotation on Field"));
-      OI.getButton(OI.Driver.orientationButton)
-          .onTrue(
-              drivetrain
-                  .runOnce(() -> drivetrain.toggleOrientation())
-                  .withName("Toggle Orientation"));
-    }
+    Supplier<DriveRequest> input =
+        () ->
+            SwerveSubsystem.joystickCondition(
+                new DriveInput(
+                    OI.getAxisSupplier(OI.Driver.xTranslationAxis).get(),
+                    OI.getAxisSupplier(OI.Driver.yTranslationAxis).get(),
+                    OI.getAxisSupplier(OI.Driver.rotationAxis).get()),
+                0.1);
 
-    // OI.Driver.getZeroButton().onTrue(new InstantCommand(() -> drivetrain.getPigeon2().reset()));
+    drivetrain.setDefaultCommand(
+        drivetrain.fieldOrientedDrive(input).withName("Get Axis Suppliers"));
 
-    // Shooter commands
-    if (Constants.enabledSubsystems.shooterEnabled && Constants.enabledSubsystems.triggerEnabled) {
-      shooterSubsystem.setDefaultCommand(shooterSubsystem.shooterIdle());
-      if (Constants.enabledSubsystems.visionEnabled) {
-        OI.getTrigger(OI.Operator.shooterRevTrigger).whileTrue(shooterSubsystem.revShooter());
-
-        OI.getTrigger(OI.Operator.shooterFireTrigger)
-            .whileTrue(
-                triggerSubsystem
-                    .getShootCommand()
-                    .onlyIf(shooterSubsystem.shooterReady())
-                    .onlyWhile(OI.getTrigger(OI.Operator.shooterRevTrigger)));
-        OI.getButton(OI.Operator.A).whileTrue(triggerSubsystem.getLoadCommand());
-      } else {
-        OI.getTrigger(OI.Operator.shooterFireTrigger).whileTrue(shooterSubsystem.bumperShoot());
-      }
-    }
+    trapElvSubsystem.setDefaultCommand(trapElvSubsystem.stowTrapElv());
```

Comment:
We removed all of the enabled/disable code.  Is this intentional?

Reply:
yes, the logic was shifted to in the subsystem, to make it easier to read how the controls flowed. Also to avoid accidental calling to a disabled subsystem without a check in the more complex logic